### PR TITLE
Fix spelling mistakes

### PR DESCRIPTION
**Pull request by jonas054**

### CODE REVIEW

File: config/default.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ AllCops:
   # -S/--display-style-guide option.
   DisplayStyleGuide: false
   # Extra details are not displayed in offense messages by default. Change
-  # behaviour by overriding ExtraDetails, or by giving the
```
Comment (maxjacobson) [inline config/default.yml]: I assume these were intentionally written with a preference for the UK spelling. I'm not sure if the codebase is consistently British, though

**Comment by bbatsov** [comment]
:+1:

File: config/default.yml (hunk 1 above)
Comment (bbatsov) [inline config/default.yml]: We decided a while back to favour American spellings in the codebase.

Comment (alexdowad) [inline config/default.yml]: You mean "favor American spellings"? :laugh:

Comment (bbatsov) [inline config/default.yml]: This was intentional. :-) I don't like American English, but I respect its dominion over codebases.

**Pull request merged**